src/components/router/Router.js

```diff
@@ -0,0 +1,26 @@
+import { renderMainPage } from '../../views/MainPage/mainPage.js';
+import { renderQuizSettings } from '../../views/quiz-settings/quiz-settings.js';
+
+const app = document.getElementById('app');
+
+const applicationRoutes = {
+  '/': renderMainPage,
+  '#quiz-settings': renderQuizSettings,
+};
+
+function setRenderFunction(path) {
+  if (applicationRoutes[path] === undefined) {
+    return () => alert('Page does not exist!');
+  } else {
+    return applicationRoutes[path];
+  }
+}
+
+export function Router() {
+  const url = window.location.hash || '/';
+  app.innerHTML = '';
+  const renderer = setRenderFunction(url);
+  if (renderer) {
+    renderer();
+  }
+}
```

Comment:
Just one idea to improve the user's experience - we need to remove this `() => alert('Page does not exist!');`, because it is still a function (so it is always true in line 23).

1st idea:
return undefined if there are no applicationRoutes[path], and add in Router() what should be done when there is no renderer

2nd idea,
prepare redirectToMainPage function and return it from setRenderFunction if there if there is no applicationRoutes[path]